packages/api/swagger-config.json

```diff
@@ -2,7 +2,7 @@
    "openapi": "3.0.2",
    "info": {
       "title": "w3name API",
-      "description": "This documentation describes the HTTP API for [**w3name**](https://name.web3.storage), which allows you to build applications that create and manage [IPNS records](https://docs.ipfs.tech/concepts/ipns/).\n\nYou can also interact with this API using client libraries. See the [**w3name Client package**](https://www.npmjs.com/package/w3name) for more information on using a client library.\n\n## API endpoint URL\n\nThe main public API endpoint URL for w3name is `https://name.web3.storage`. All endpoints described in this document should be made relative to this root URL. For example, to post to the `/name/:key` endpoint, send your request to `https://name.web3.storage/name/:key`.",
+      "description": "This documentation describes how to use our HTTP API for w3name.\n\nThis API will allow you to build applications that create and manage IPNS records. IPNS is short for InterPlanetary Name System and adds a new layer to the powerful content addressing that IPFS provides. w3name is a service that provides secure, stable identifiers for content addressable and mutable data (that changes over time).\n\nWhile you can interact with this API in a simple way using our JS client library, we also provide this API on its own for you to leverage in your application.\n\nThis API will allow you to create, update and retrieve a w3name record. However, in order to do this, you will first need to create key pair. You can do this for example through the IPNS CLI ([documentation here](https://docs.ipfs.tech/concepts/ipns/#example-ipns-setup-with-cli)).\n\n\nOnce the keypair is created you can use the public key in our w3name API payload to create a new w3name record and publish it to the DHT.\n\n\n\n## API endpoint URL\n\nThe main public API endpoint URL for w3name is `https://name.web3.storage`. All endpoints described in this document should be made relative to this root URL. For example, to post to the `/name/:key` endpoint, send your request to `https://name.web3.storage/name/:key`.",
```

Comment:
* "create key pair" -> "create a keypair"
* Pick a convention: key pair or keypair?
* Once the keypair is created you can use the public key in our w3name API payload to create a new w3name record and publish it to the DHT. -> Once you have created a keypair, you can create and sign an IPNS record that you will be able to use with our w3name API endpoint for publishing onto the DHTs.

Reply:
Thanks! This makes so much more sense!